app/views/admin/user.scala.html

```diff
@@ -77,6 +77,56 @@ <h1>General Info and Stats</h1>
             </div>
         </div>
 
+        <h1>Mark Work Quality by Date</h1>
+        <p>
+            Select a date to mark all labels for this user before that date with the desired flag. This is useful when
+            training new users on Project Sidewalk. If their work was low quality, set it as such below. Their labels
+            will then be partially hidden in our APIs, and it won't count against their future accuracy rating.
+        </p>
+        <p>
+            If their work was high quality, but they did not use all the available label types or didn't label the
+            entire street, you can mark their data as "incomplete". Their labels will show up in our APIs, but we'll
+            have other users re-audit those streets.
+        </p>
+        <div class="row set-flags-container">
+            <div class="col-lg-4 flag-date-container">
+                <h2>Low Quality</h2>
+                <div class="row">
+                    <div class="input-group date datepicker" data-date-format="mm-dd-yyyy">
+                        <input id="low-quality-date" class="form-control" type="text"/>
+                        <span class="input-group-addon">
+                            <i class="glyphicon glyphicon-calendar"></i>
+                        </span>
+                    </div>
+                </div>
+                <div class="row date-submit-row">
+                    <button type="button" class="btn btn-primary date-submit" onclick=setLowQualityDate(true)>Set Flags</button>
+                    <button type="button" class="btn btn-primary date-submit" onclick=setLowQualityDate(false)>Remove Flags</button>
+                </div>
+                <div class="row">
+                    <div id="low-quality-alert" class="alert alert-success" role="alert" style="visibility: hidden">Alert TEST</div>
+                </div>
+            </div>
+            <div class="col-lg-4 flag-date-container">
+                <h2>Incomplete</h2>
+                <div class="row">
+                    <div class="input-group date datepicker" data-date-format="mm-dd-yyyy">
```

Comment:
How easy is it to include time instead of just the date? Though that isn't as important as making sure that by default, data that was contributed today is included. Right now it marks everything before 12:00 am today by default, which essentially means that it defaulted to yesterday :grin: